go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/submariner-io/admiral v0.9.0-rc0
 	github.com/submariner-io/cloud-prepare v0.0.0-20210422124637-01644d90a37b
 	github.com/submariner-io/lighthouse v0.9.0-rc0
-	github.com/submariner-io/shipyard v0.9.0-rc0
+	github.com/submariner-io/shipyard v0.9.0-rc0.0.20210428164906-1a52a9816269
```

Comment:
This is still unmerged, let's reset go.mod & go.sum  :) we can grab those changes once merged.